src/main/java/oracle/r2dbc/impl/OracleReactiveJdbcAdapter.java

```diff
@@ -432,9 +341,40 @@ public DataSource createDataSource(ConnectionFactoryOptions options) {
   }
 
   /**
+   * <p>
    * Composes an Oracle JDBC URL from {@code ConnectionFactoryOptions}, as
    * specified in the javadoc of
    * {@link #createDataSource(ConnectionFactoryOptions)}
+   * </p><p>
+   * If the {@link ConnectionFactoryOptions#SSL} option is set, then the JDBC
+   * URL is composed with the tcps protocol, as in:
+   * {@code jdbc:oracle:thins:@tcps:...}. The {@code SSL} option is interpreted
```

Comment:
What is "thins" in "jdbc:oracle:thins:@..."?

Reply:
This was a typo. I've fixed it now.